site/content/posts/2023-08-08-v0.25.0-impersonation-proxy-with-external-certs.md

```diff
@@ -0,0 +1,318 @@
+---
+title: "Pinniped v0.25.0: With External Certificate Management for the Impersonation Proxy and more"
```

Comment:
I like the title, but since we don't have a community manager I'm guessing we probably need to do our own SEO.  We should likely add the words "Kubernetes", "authentication", and "OIDC" in the title and page content if possible.  Human readable of course, doesn't have to be chunky sounding.

Reply:
Could be helpful to include PKI even, since this is about public key infrastructure.  That could be a "long tail" kind of keyword that would cause someone to find this document.

IE, I imagine someone would search for something like "Solution for Kubernetes Authentication with an OIDC Identity Provider with my own certificates or PKI" ~ or something along these lines.

Reply:
I'll add `'kubernetes'`, `'pki'`, `'pinniped'` to the list of `tags` for the blog post.

Reply:
Nice.